src/main.h

```diff
@@ -144,6 +146,8 @@ typedef struct
   cairo_surface_t *undobuffer[GROMIT_MAX_UNDO];
   gint            undo_head, undo_depth, redo_depth;
 
+  cairo_surface_t *temp_buffer;
```

Comment:
Maybe a more speaking name or a comment above?

Reply:
To me it seems it's the back buffer of the back buffer, but this might not be a good name...